sopel/config/types.py

```diff
@@ -236,17 +266,22 @@ def configure(self, prompt, default, parent, section_name):
             prompt = prompt[0]
 
         if default is not NO_DEFAULT:
-            default = ','.join(default)
-            prompt = '{} [{}]'.format(prompt, default)
+            default_prompt = ','.join(map(lambda item: '"{}"'.format(item), default))
+            prompt = '{} [{}]'.format(prompt, default_prompt)
         else:
-            default = ''
+            default = []
         print(prompt)
         values = []
         value = get_input(each_prompt + ' ') or default
+        if (value == default) and len(default) == 0:
+            value = ''
         while value:
-            values.append(value)
+            if (value == default):
```

Comment:
You don't need the extra parenthesis here.

Reply:
:+1: